circleci/catapult

```diff
@@ -4,21 +4,27 @@
 #
 # Usage:
 #
-#   circleci-catapult [CATAPULT_URL] [APP_NAME]
+#   circleci-catapult [APP_NAME]
 
-# TODO: Implement functionality from here...
-# https://github.com/Clever/drone/blob/master/plugin/publish/catapult.go
 set -e
 
 # User supplied args
-CATAPULT_URL=$1
-if [[ -z $CATAPULT_URL ]]; then echo "Missing arg1 CATAPULT_URL" && exit 1; fi
-APP_NAME=$2
+APP_NAME=$1
 if [[ -z $APP_NAME ]]; then echo "Missing arg2 APP_NAME" && exit 1; fi
 
 # Set automatically by CircleCI
 : ${CIRCLE_PROJECT_REPONAME?"Missing required env var"}
 REPO=$CIRCLE_PROJECT_REPONAME
+: ${CIRCLE_PROJECT_USERNAME?"Missing required env var"}
+USER=$CIRCLE_PROJECT_USERNAME
+: ${CIRCLE_BUILD_NUM?"Missing required env var"}
+BUILD_NUM=$CIRCLE_BUILD_NUM
 
-echo "Error! Not yet implemented"
-return 1
+# Should be in CircleCI
+: ${CATAPULT_URL?"Missing required env var"}
+CATAPULT_URL=$CATAPULT_URL
+: ${CATAPULT_BASIC_AUTH?"Missing required env var"}
+BASIC_AUTH=$CATAPULT_BASIC_AUTH
```

Comment:
What do you think makes the most sense?

(2) would be most equivalent to how we've done it for other commands; but we're still pretty early so we could rethink this if there's a most convenient approach.